ermia_takada/frame.cc

```diff
@@ -370,6 +372,12 @@ int main(int argc, char *argv[])
     }
 
     print_mode();
+
+    // TMT = new TransactionTable *[thread_num];
+
+    for (int i = 0; i < thread_num; ++i)
+        TMT[i] = new TransactionTable(0, 0, UINT32_MAX, 0, Status::inFlight);
```

Comment:
論文では
```C++
u = get_transaction(v_sstamp)
```
となっており，文中では

> it will obtain U’s transaction context (line 10) through a centralized transaction table indexed by TIDs

とありますが，今回TXIDではなくスレッドIDをキーにしたテーブルになっていることで問題はないのでしょうか？

たとえば，あるスレッドAが スレッドBで実行されているTXID=1 について
```c++
# this->thid_ == A
# tmt.thid_ == B
while(tmt.cstamp.load() == 0)
```
というふうなループを回しているとして，もしAがプリエンプションしている間にBがコミットまで完了しきって，新しいトランザクションを開始したらどうなりますか？バリデーションのロジックが壊れるような気もします．